doc/user/zebra.rst

```diff
@@ -31,6 +31,15 @@ Besides the common invocation options (:ref:`common-invocation-options`), the
 
    When program terminates, retain routes added by zebra.
 
+.. option:: -n --vrfwnetns
+
+   When zebra starts with this option, the VRF backend is
+   based on `Linux network namespaces`. That implies that all
+   network namespaces discovered by ZEBRA will create an
+   associated VRF. The other daemons will operate on
+   the VRF defined by ZEBRA, as usual. More information on
+   VRF Commands chapter.
```

Comment:
Please put a cross-reference to the section, like so:
```
...More information in the :ref:`vrf-commands` section.
```
The result will be the title of the section (VRF Commands) hyperlinked to that section.
http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-arbitrary-locations

Also, "More information *in *the VRF Commands chapter"

Reply:
thanks for help.